### DISCUSSION TITLE
Does unocss support SSR?

### DISCUSSION THREAD

**Discussion by huage404** [unocss/unocss#2712] [Q&A]
I imported unocss/nuxt@^0.52.3 into my nuxt@^2.16.3 project, and it worked fine in my local development environment. However, when I deployed it to the server, all the configurations in my uno.config.js did not take effect.

**Comment by serkodev** [comment]
I think UnoCSS support SSR, check more in this project: https://github.com/elk-zone/elk

If possible, could you please share your source code? This would make it much easier for everyone to understand the details of your issue.
